toutvcli/app.py

```diff
@@ -87,6 +88,13 @@ def run(self):
 
         self._verbose = args.verbose
 
+        if 'quiet' in args:
```

Comment:
Since this is a switch specific to the `fetch` command, this code should probably go in the `command_fetch` method.